src/Resources/TransactionTypes/Payout.php

```diff
@@ -24,14 +24,7 @@ class Payout extends AbstractTransactionType
     /** @var string $paymentReference */
     protected $paymentReference;
 
-    /**
-     * Payout constructor.
-     *
-     * @param float|null  $amount
-     * @param string|null $currency
-     * @param null        $returnUrl
-     */
-    public function __construct(float $amount = null, string $currency = null, $returnUrl = null)
+    public function __construct(?float $amount = null, ?string $currency = null, $returnUrl = null)
```

Comment:
I do not think we should remove information, but rather add it.
AFAIK the phpdoc information is also used for code completion purposes.
It is also relevant for AI to better understand the context of the code.